src/Build.UnitTests/BackEnd/MSBuild_Tests.cs

```diff
@@ -69,15 +68,15 @@ public void ProjectItemSpecTooLong()
                 }
 
                 int rootLength = Path.GetPathRoot(tempPath).Length;
-                string tempPathNoRoot = tempPath.Substring(rootLength);
+                string tempPathNoRoot = tempProject.Substring(rootLength);
```

Comment:
Will it be possible to get the length of the rootPath from the tempProject instead of tempPath? the reasoning behind that is: 
1 - In case the tempPath root length would differ from the root length of projectPath

what do you think?

Reply:
Thanks for your suggestions. Have updated